src/js/main.js

```diff
@@ -155,6 +151,25 @@ var SupportKit = Marionette.Object.extend({
             .done();
     },
 
+    logout: function() {
+        this.destroy();
+    },
+
+    getDeviceId: function(options) {
+        var userId = options.userId, deviceId;
+
+        // get device ID first from local storage, then cookie. Otherwise generate new one
+        deviceId = userId && localStorage.getItem(SK_STORAGE + '_' + userId) ||
+            cookie.parse(document.cookie)[SK_STORAGE] ||
+            uuid.v4().replace(/-/g, '');
+
+        // reset the cookie and local storage
+        document.cookie = SK_STORAGE + '=' + deviceId;
+        userId && localStorage.setItem(SK_STORAGE + '_' + userId, deviceId);
+
+        return deviceId;
+    },
+
     resetUnread: function() {
         this._checkReady();
         this._chatController._resetUnread();
```

Comment:
I guess `_resetUnread` in the chatController should be `resetUnread` if it's called from here.